_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2022-01-06
+categories: [team, engineering]
+author: artsy_engineering
+---
+
+Ufff, 2021 was a tough year. It was the second year of the covid-19 pandemic. We tried our best to support each
+other through these challenging times and in that spirit, we feel like it's important not to lose sight of our
+accomplishments. As we say goodbye to 2021 and hello to 2022, here are some Artsy Engineering wins from this past year.
+What did you ship in 2021? We would love to hear about it and celebrate with you!
```

Comment:
Given this CTA (which I like very much), we should [add a comment thread for the article](https://github.com/artsy/artsy.github.io#enabling-comments)

Reply:
Added! 👍🏼